tests/src/test/scala/pureconfig/ConfigSourceSuite.scala

```diff
@@ -0,0 +1,205 @@
+package pureconfig
+
+import java.io.IOException
+
+import com.typesafe.config.{ Config, ConfigFactory, ConfigValueType }
+import pureconfig.PathUtils._
+import pureconfig.error._
+import pureconfig.generic.auto._
+
+class ConfigSourceSuite extends BaseSuite {
+
+  behavior of "ConfigSource"
+
+  it should "load from reference.conf by default" in {
+    case class Conf(d: Double, i: Int, s: String)
+    ConfigSource.default.load[Conf] shouldBe Right(Conf(0D, 0, "default"))
```

Comment:
Can we also add an `application.conf` file to the test resources to show that it takes precedence over the `reference.conf` in `ConfigSource.default`?

Reply:
👍